### PR TITLE
Fix truncateString

### PR DESCRIPTION
**Pull request by emilie-blackbird**
### Description (*)
Fix truncateString method :  "$_remainder must be of the type string, null given"

### Fixed Issues (if relevant)
https://github.com/magento/magento2/issues/16958 : Order View Issue - This tab contains invalid data

### Manual testing scenarios (*)
1. Go to admin
2. Open a specific order in sales
3.  Some Order have the error "This tab contains invalid data"

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=20849) <br/>All committers have signed the CLA.

**Comment by magento-engcom-team** [comment]
Hi @emilie-blackbird. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.2-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Sales/Block/Adminhtml/Items/Column/Name.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function truncateString(
         string $value,
         int $length = 80,
         string $etc = '...',
-        string &$remainder = '',
+        ?string &$remainder = '',
```
Comment (orlangur) [inline app/code/Magento/Sales/Block/Adminhtml/Items/Column/Name.php]: Not acceptable as PHP 7.0 is supported in 2.2.

Is this fixed in 2.3?

**Review by orlangur — CHANGES_REQUESTED** [review]

File: app/code/Magento/Sales/Block/Adminhtml/Items/Column/Name.php (hunk 1 above)
Comment (emilie-blackbird) [reply app/code/Magento/Sales/Block/Adminhtml/Items/Column/Name.php]: In 2.3 it isn't in strict typing.

**Comment by sidolov** [comment]
Hi @emilie-blackbird  , looks like you made some commits with email different than in your GitHub profile, please, add email from commits to your profile!
Thank you!

**Review by orlangur — CHANGES_REQUESTED** [review]
Please squash changes into a single commit so that we have perfectly clean history 😉

**Comment by magento-engcom-team** [comment]
@emilie-blackbird thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by p-bystritsky** [comment]
@orlangur PR updated, please review.

**Review by orlangur — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @orlangur, thank you for the review.
[ENGCOM-5211](https://jira.corp.magento.com/browse/ENGCOM-5211) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
@emilie-blackbird thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by VasylShvorak** [comment]
:heavy_check_mark: QA passed

**Comment by SPbryan007** [comment]
@magento give me 2.4.5-p1 instance

**Pull request merged**